src/task/task.json

```diff
@@ -55,6 +55,15 @@
             },
             "groupName": "configurationOptions"
         },
+        {
+            "name": "OutputIssuesWithoutFailing",
```

Comment:
I'd rather call this "Fail build on code issues" which is on by default. This sounds more logical in my opinion